docs/site/Polymorphic-relations.md

```diff
@@ -3,7 +3,7 @@ lang: en
 title: 'Polymorphic Relations'
 keywords: LoopBack 4.0, LoopBack 4, Node.js, TypeScript, OpenAPI, Model Relation
 sidebar: lb4_sidebar
-permalink: /doc/en/lb4/Polymorphic-relation.html
+permalink: /doc/en/lb4/Polymorphic-relations.html
```

Comment:
The other relation pages are called `-relation.html`. For consistency, can we stick to the singular form?

```suggestion
permalink: /doc/en/lb4/Polymorphic-relation.html
```

Reply:
That's ok. In loopback 3 they were plural, but seems in lb4 they are all changed to singular. I will also change the file name to the singular form.